src/RobotInterface/YarpImplementation/include/BipedalLocomotion/RobotInterface/YarpSensorBridgeImpl.h

```diff
@@ -824,7 +727,7 @@ struct YarpSensorBridge::Impl
                 return false;
             }
         }
-
+        
```

Comment:
White spaces

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/182/commits/d4dcb8a28a7f4b4a59fee3f7b96e2f25f6a4d268.